drivers/scheduler/k8s/k8s.go

```diff
@@ -262,6 +271,15 @@ func (k *K8s) ParseSpecs(specDir string) ([]interface{}, error) {
 	return specs, nil
 }
 
+func isValidProvider(specPath string) bool {
+	for provider := range spec.SupportedStorageProviders {
+		if strings.Contains(specPath, provider) {
```

Comment:
Should check for the driver name being the suffix, this will match the driver anywhere in the path